docs/docs/api/endpoints/components.md

```diff
@@ -112,3 +112,27 @@ The list of the deleted components is returned.
   }
 }
 ```
+
+## List
+
+Delete a list of component which are currently deployed.
+
+`POST /components.list`
+
+**Sample request**
```

Comment:
If the request is with an empty payload, all the components from all the repositories should be returned.